src/test/frameworks/compose/discover/filesystem.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { RepositoryFileSystem } from "../../../../frameworks/compose/discover/filesystem";
+import { expect } from "chai";
+
+describe("RepositoryFileSystem", () => {
+  let fileSystem: RepositoryFileSystem;
+
+  before(() => {
+    fileSystem = new RepositoryFileSystem("./src/frameworks/compose/discover/testapps/expressApp");
```

Comment:
as mentioned in another comment:

Consider using a in memory filesystem instead.